.idea/.generators

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
`.idea` klasorunu git'e eklemeyin lutfen :)  http://stackoverflow.com/questions/8527597/how-do-i-ignore-files-in-a-directory-in-git